source/sgp_mode/Tasks.h

```diff
@@ -154,6 +179,13 @@ class TaskSet {
           score = MarkPerformedTask(state, i, shared, score);
           state.internalEnvironment->insert(state.internalEnvironment->begin(),
                                             sqrt(output));
+          if(state.host->IsHost()){
+            task.curHostOutput = output;
+            IncrementSquareMap(task, state, output, task.hostCalculationTable);
```

Comment:
Should probably only record this information if you are doing square tasks so that there isn't extra tracking when they aren't doing squares